opencti-platform/opencti-graphql/src/domain/stixRelationship.js

```diff
@@ -62,16 +65,24 @@ export const stixRelationshipDelete = async (context, user, stixRelationshipId)
   return stixRelationshipId;
 };
 
+const buildValidWidgetRelationshipType = (relationship_type) => {
+  const isValidRelationshipType = isStixCoreRelationship(relationship_type)
+    || isStixSightingRelationship(relationship_type)
+    || relationship_type === RELATION_CONTAINS;
+  return isValidRelationshipType ? relationship_type : widgetsRelationshipTypes;
```

Comment:
it can be both a string and an array of strings, is it ok ?

Reply:
it's not ok indeed...

Reply:
fixed :)